$NetBSD: patch-CMakeLists.txt,v 1.3 2022/09/24 08:46:23 nia Exp $

Do not force libstdc++ to be linked statically, creates problems
with PIE code.

--- CMakeLists.txt.orig	2022-09-24 08:40:00.299506879 +0000
+++ CMakeLists.txt
@@ -607,10 +607,6 @@ foreach(target devilution ${BIN_TARGET} 
   target_compile_definitions(${target} PRIVATE ${def_list})
 endforeach(target)
 
-if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND NOT HAIKU AND NOT VITA)
-  target_link_libraries(${BIN_TARGET} PUBLIC "$<$<NOT:$<CONFIG:Debug>>:-static-libgcc;-static-libstdc++>")
-endif()
-
 if(WIN32)
   target_link_libraries(${BIN_TARGET} PRIVATE wsock32 ws2_32 wininet)
 
